### PR TITLE
[2018-10-02] [bRO] Tables update

### PR DESCRIPTION
**Pull request by lututui**

### CODE REVIEW

File: tables/bRO/itemslotcounttable.txt [hunk 1]
```diff
@@ -1179,7 +1179,6 @@
 19315#1#
 19316#1#
 19326#1#
-20266#1#
```
Comment (sctnightcore) [inline tables/bRO/itemslotcounttable.txt]: :thinking:

Comment (alisonrag) [reply tables/bRO/itemslotcounttable.txt]: some items show wrong slots number in bRO, maybe this is removing because this item dont have slot

Comment (sctnightcore) [reply tables/bRO/itemslotcounttable.txt]: :ok_hand:

**Comment by dimas2300** [comment]
доброго времени суток прозьба есть помогите с подключение на  4game если можите

**Pull request merged**